spec/requests/user_subscriptions_spec.rb

```diff
@@ -140,13 +137,12 @@
       end.to change(UserSubscription, :count).by(0)
 
       expect(response).to have_http_status(:unprocessable_entity)
-      expect(response.parsed_body["error"]).to eq("subscriber email mismatch")
+      expect(response.parsed_body["error"]).to eq("Subscriber email mismatch.")
```

Comment:
For these error checks it might be easier to do `include("email mismatch")` so if you tweak the wording you dont have to always update these specs.

Reply:
Good idea - updated!